climada/hazard/tc_tracks_synth.py

```diff
@@ -166,6 +206,170 @@ def _one_rnd_walk(track, ens_size, ens_amp0, ens_amp, max_angle, rnd_vec):
     return ens_track
 
 
+def _random_uniform_ac(n_ts, autocorr, time_step_h):
+    """Generate a series of autocorrelated uniformly distributed random numbers.
+
+    This implements the algorithm described here to derive a uniformly distributed
+    series with specified autocorrelation (here at a lag of 1 hour):
+    https://stats.stackexchange.com/questions/48086/
+        algorithm-to-produce-autocorrelated-uniformly-distributed-number
```

Comment:
Why was this algorithm preferred to the one from the there mentioned paper [A method to generate autocorrelated uniform random numbers](https://www.tandfonline.com/doi/abs/10.1080/00949659308811469)?
As I understand it, it's only an approximation while the paper's method is seemingly a correct solution and looks like it would run much faster.
Apart from that, it has the benefit, that the reference to a paper is there to stay and its content cannot be removed or edited at any time in the future.

Reply:
Agreed. Must not necessarily be part of this pull request. We can come back to it later.